annotation/src/main/java/org/cbioportal/genome_nexus/annotation/domain/EnsemblTranscript.java

```diff
@@ -0,0 +1,66 @@
+package org.cbioportal.genome_nexus.annotation.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.univocity.parsers.annotations.Parsed;
+import com.univocity.parsers.annotations.Trim;
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * Ensembl transcript
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Ideally we should add API related annotations like `@JsonInclude` and `@ApiModelProperty` in a mixin under web package, but we can also do that in another PR.